core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -94,5 +98,18 @@ var ExplorationEditorTranslationTab = function() {
     _selectLanguage(language);
     waitFor.pageToFullyLoad();
   };
+
+  this.navigateToTranslationTabFeedback = function() {
```

Comment:
This file represents the TranslationTab, so you don't have to add "TranslationTab" here, make it `navigateToFeedbackTab`.

Reply:
done